get.sh

```diff
@@ -98,49 +98,42 @@ parseCommandLineArgs()
 
 getBinaryOpenjdk()
 {
+	echo "get jdk binary..."
 	cd $SDKDIR
 	mkdir openjdkbinary
 	cd openjdkbinary
-	jar_file_name=""
-	if [[ "$CUSTOMIZED_SDK_URL" == "" ]]; then
-		if [[ "$SDK_RESOURCE" == "nightly" || "$SDK_RESOURCE" == "releases" ]]; then
-			echo 'Get binary openjdk...'
-			download_url="https://api.adoptopenjdk.net/$JVMVERSION/$SDK_RESOURCE/$PLATFORM/latest/binary"
-			curl -OLJ $download_url
-			jar_file_name=`ls`
-			if [ "$jar_file_name" = "binary" ]; then
-				echo "No available jdk binary, exiting"
-				exit 1
-			fi
-		fi
-	else 
+	
+	if [ "$CUSTOMIZED_SDK_URL" != "" ]; then
 		download_url=$CUSTOMIZED_SDK_URL
-		if [[ "$PLATFORM" == "s390x_zos" ]];
-		then
-			curl -O ${download_url} 
-		else
-			wget -q --no-check-certificate --header 'Cookie: allow-download=1' ${download_url}
-		fi
+	elif [ "$SDK_RESOURCE" == "nightly" ] || [ "$SDK_RESOURCE" == "releases" ]; then
+		download_url="https://api.adoptopenjdk.net/$JVMVERSION/$SDK_RESOURCE/$PLATFORM/latest/binary"
+	else
+		download_url=""
+		echo "--sdkdir is set to $SDK_RESOURCE. Therefore, skip download jdk binary"
+	fi
+	
+	if  [ "${download_url}" != "" ]; then
+		echo "curl -OLJks "${download_url}""
+		curl -OLJks "${download_url}"
 		if [ $? -ne 0 ]; then
```

Comment:
This check may not work for curl -O options. That is saying when curl -OLJ fails to download SDK the return CODE is still 0.